src/QsCompiler/Tests.Compiler/ExecutionTests.fs

```diff
@@ -40,7 +40,7 @@ type ExecutionTests (output:ITestOutputHelper) =
         AssertEqual expectedOutput out
 
 
-    [<Fact>]
+    //[<Fact>]
```

Comment:
For visibility, please mark the test as 'Skipped' and provide a short reason instead of commenting it out.

Reply:
This is temporary and will be reverted shortly.